iOS-sample-app/Showcases/Screen Views/TrackedViewControllers.swift

```diff
@@ -0,0 +1,29 @@
+import UIKit
+import ContentSquare
+
+// Use this custom class for view controllers in your storyboard, and they will start sending screen views. They
+// will still be editable in the storyboard as usual. You can use the same approach for any cotroller which you edit in
+// a storyboard and for which you don't have code.
+class TrackedViewController: UIViewController {
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        sendTitleScreenview()
+    }
+}
+
+// In the same way, use this custom class for table view controllers in your storyboard, and they will start sending
+// screen views. Again, they will still be editable in the storyboard as usual table view controllers.
+class TrackedTableViewController: UITableViewController {
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        sendTitleScreenview()
+    }
+}
+
+extension UIViewController {
```

Comment:
I actually think that's a pretty bad idea. That's how it's done in our SDK and it's made me struggle so much finding how things are implemented. We have a ViewControllerExtension.swift file which contains a way too long bunch of completely unrelated extensions, instead of having each one along the rest of the business code it's related to.

I moved things a bit but in a way I hope makes more sense. See [here](https://github.com/ContentSquare/iOS-sample-app/pull/2/commits/436e63ccaead04a871354383c17c09e51d0ce877) and [here](https://github.com/ContentSquare/iOS-sample-app/pull/2/commits/d51d9328c40bcec9a49ceaebd7946311ae053a54).